testcafe/check-testcafe-results.js

```diff
@@ -47,7 +47,7 @@ If they seem to be failing unexpectedly, check grafana for ingestion lag at http
     log(JSON.stringify(files, null, 2))
 
     // the deadline is the same for each assert, as the ingestion lag will be happening in parallel
-    const deadline = Date.now() + 1000 * 60 * 30 // 30 minutes
+    const deadline = Date.now() + 1000 * 60 * 120 // 30 minutes
```

Comment:
pedantic, but you change the comment to match the time (it's 120 minutes now).

Reply:
I was hoping to not leave this around, but may do so depending on what others think

Reply:
yeah fair 'nuff